app/uk/gov/hmrc/tariffclassificationfrontend/models/Permission.scala

```diff
@@ -23,20 +23,20 @@ object Permission extends Enumeration {
 
   val CREATE_CASES, VIEW_MY_CASES, VIEW_QUEUE_CASES, VIEW_ASSIGNED_CASES, SEARCH, ADVANCED_SEARCH, VIEW_CASES, VIEW_CASE_ASSIGNEE, ASSIGN_CASE,
   RELEASE_CASE, SUPPRESS_CASE, REFER_CASE, REOPEN_CASE, REJECT_CASE, SUSPEND_CASE, COMPLETE_CASE, REMOVE_ATTACHMENTS,
-  CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE,
+  CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_LIABILITY, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE,
   MANAGE_KEYWORDS, MANAGE_QUEUES, VIEW_REPORTS, EDIT_SAMPLE = Value
 
   val teamCaseOwnerPermissions: Set[Permission] = Set(CREATE_CASES, VIEW_MY_CASES, VIEW_QUEUE_CASES, SEARCH, ADVANCED_SEARCH, VIEW_CASES, ASSIGN_CASE,
     RELEASE_CASE, SUPPRESS_CASE, REFER_CASE, REOPEN_CASE, REJECT_CASE, SUSPEND_CASE, COMPLETE_CASE, REMOVE_ATTACHMENTS,
-    CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE,EDIT_SAMPLE)
+    CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_LIABILITY, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE,EDIT_SAMPLE)
 
   val readOnlyPermissions: Set[Permission] = Set(SEARCH, ADVANCED_SEARCH, VIEW_CASES)
 
   val systemAdminPermissions: Set[Permission] = Set(MANAGE_KEYWORDS, MANAGE_QUEUES)
 
   private val managerPermissions: Set[Permission] = Set(CREATE_CASES, VIEW_MY_CASES, VIEW_QUEUE_CASES, VIEW_ASSIGNED_CASES, SEARCH, ADVANCED_SEARCH, VIEW_CASES, ASSIGN_CASE, VIEW_CASE_ASSIGNEE,
     RELEASE_CASE, SUPPRESS_CASE, REFER_CASE, REOPEN_CASE, REJECT_CASE, SUSPEND_CASE, COMPLETE_CASE, REMOVE_ATTACHMENTS,
-    CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE, VIEW_REPORTS, EDIT_SAMPLE)
+    CANCEL_CASE, ADD_NOTE, ADD_ATTACHMENT, KEYWORDS, EDIT_LIABILITY, EDIT_RULING, APPEAL_CASE, EXTENDED_USE, MOVE_CASE_BACK_TO_QUEUE, VIEW_REPORTS, EDIT_SAMPLE)
 
   private val teamBasicPermissions: Set[Permission] = Set(CREATE_CASES, VIEW_MY_CASES, VIEW_QUEUE_CASES, SEARCH, ADVANCED_SEARCH, VIEW_CASES, ASSIGN_CASE, VIEW_CASE_ASSIGNEE,
```

Comment:
There is an special case for this edit liability. If the case is NEW anyone (OWNER, NOT-OWNER, ADMIN) can edit the liability.

Reply:
I'll relax the  controller validation so it just checks the case status. I'll implement the proper rule on the edit button

Reply:
Done

Reply:
Looks good!